src/main/java/net/macmv/rgen/item/RItems.java

```diff
@@ -49,6 +51,31 @@ public class RItems {
   public static final Item BASALT = registerBlockItem(RBlocks.BASALT);
   public static final Item PINK_FLOWERBED = registerBlockItem(RBlocks.PINK_FLOWERBED);
 
+  public static final Item DEAD_DOOR = registerItem("dead_door", new ItemDoor(RBlocks.DEAD_DOOR));
+  public static final Item MANGROVE_DOOR = registerItem("mangrove_door", new ItemDoor(RBlocks.MANGROVE_DOOR));
+  public static final Item BAMBOO_DOOR = registerItem("bamboo_door", new ItemDoor(RBlocks.BAMBOO_DOOR));
+  public static final Item CEDAR_DOOR = registerItem("cedar_door", new ItemDoor(RBlocks.CEDAR_DOOR));
+  public static final Item CHERRY_DOOR = registerItem("cherry_door", new ItemDoor(RBlocks.CHERRY_DOOR));
+  public static final Item FIR_DOOR = registerItem("fir_door", new ItemDoor(RBlocks.FIR_DOOR));
+  public static final Item PALM_DOOR = registerItem("palm_door", new ItemDoor(RBlocks.PALM_DOOR));
+
+  public static final Item DEAD_STAIRS = registerItem("dead_stairs", new ItemBlock(RBlocks.DEAD_STAIRS));
+  public static final Item MANGROVE_STAIRS = registerItem("mangrove_stairs", new ItemBlock(RBlocks.MANGROVE_STAIRS));
+  //public static final Item BAMBOO_STAIRS = registerItem("bamboo_stairs", new ItemBlock(RBlocks.BAMBOO_STAIRS));
+  public static final Item CEDAR_STAIRS = registerItem("cedar_stairs", new ItemBlock(RBlocks.CEDAR_STAIRS));
+  public static final Item CHERRY_STAIRS = registerItem("cherry_stairs", new ItemBlock(RBlocks.CHERRY_STAIRS));
+  public static final Item FIR_STAIRS = registerItem("fir_stairs", new ItemBlock(RBlocks.FIR_STAIRS));
+  public static final Item PALM_STAIRS = registerItem("palm_stairs", new ItemBlock(RBlocks.PALM_STAIRS));
+
+  public static final Item DEAD_TRAPDOOR = registerItem("dead_trapdoor", new ItemBlock(RBlocks.DEAD_TRAPDOOR));
+  public static final Item MANGROVE_TRAPDOOR = registerItem("mangrove_trapdoor", new ItemBlock(RBlocks.MANGROVE_TRAPDOOR));
+  //public static final Item BAMBOO_TRAPDOOR = registerItem("bamboo_trapdoor", new ItemBlock(RBlocks.BAMBOO_TRAPDOOR));
+  public static final Item CEDAR_TRAPDOOR = registerItem("cedar_trapdoor", new ItemBlock(RBlocks.CEDAR_TRAPDOOR));
+  public static final Item CHERRY_TRAPDOOR = registerItem("cherry_trapdoor", new ItemBlock(RBlocks.CHERRY_TRAPDOOR));
+  public static final Item FIR_TRAPDOOR = registerItem("fir_trapdoor", new ItemBlock(RBlocks.FIR_TRAPDOOR));
+  public static final Item PALM_TRAPDOOR = registerItem("palm_trapdoor", new ItemBlock(RBlocks.PALM_TRAPDOOR));
```

Comment:
stairs and trapdoors should be registered with `registerBlockItem`, not `registerItem`.

Reply:
485b8d30ab4a547b61caefdb359946de5f08bb33 resolved